test/test_util.py

```diff
@@ -115,7 +115,7 @@ def popen_fail(*args, **kwargs):
         with self.assertRaises(subprocess.CalledProcessError) as exc_context:
             util.command_output([b"taga", b"\xc3\xa9"])
         self.assertEqual(exc_context.exception.returncode, 1)
-        self.assertEqual(exc_context.exception.cmd, b"taga \xc3\xa9")
+        self.assertEqual(exc_context.exception.cmd, 'taga \xc3\xa9')
```

Comment:
I'm actually not quite sure. It would be one thing if this were a straightforward "can't compare bytes and unicode" issue, but something a bit subtler is going on here with the encoding and re-decoding so that we're actually getting different mojibake. I'll have to ponder this—maybe it would (paradoxically) be more straightforward to just use bytes on Unix, even on Python 3? (That might also be a terrible idea!)